### ISSUE TITLE
Reload old studies.

### ISSUE THREAD

**Issue by cleme** [clemente-lab/mmeds-meta#353]
**Is your feature request related to a problem? Please describe.**
When the DB is wiped, previous studies are not uploaded again.

**Describe the solution you'd like**
A semi-automated solution to re-upload existing studies should be in place. TBD how to implement.

**Comment by adamcantor22** [comment]
Sub-issue of #385

**Comment by adamcantor22** [comment]
Perhaps this issue deserves more priority? With the changes we're planning to make to the databases (and possibly schema) it seems likely that at some point we'll need to reload the database, so it could be good to have a plan in place for what happens before those changes are made. 

The crux of this issue seems to be: when wiping the database, all *users* are wiped and have to be registered again. But, all studies have to be associated with a user. So, perhaps one of a few things: we either figure out a way to reload both studies *and* users as part of this issue, or we just reload the studies, but make them "public" so any user can see them, or we find a way to make those studies populate to a particular user id when that user gets re-registered.

All of those solutions sound a little wonky to me, thoughts on this @cleme ?

**Comment by cleme** [comment]
The question would be: why do users need to be wiped out when we wipe the DB? Making all studies public is not an option, because some studies might require to be kept private (think: unpublished data).

It makes sense that a study always belongs to a user, but if we can keep user info *independently* of the studies, then that would be the way to go. Having said that, I consider this a lower priority issue since I think reorganizing needs to happen first, then we think about how to load older studies under this new organization (all of this assuming users can be kept when wiping the DB).

**Comment by adamcantor22** [comment]
Keeping this issue open for future necessary reuploads. Current reuploads that aren't being done yet are:
-MECONIUM
-IgA seq studies

**Comment by adamcantor22** [comment]
As the main reload still needed is MECONIUM, which is predicated on the completion of #407 from release 0.9.0, moving to 0.9.0.
